rotkehlchen/globaldb/manual_price_oracles.py

```diff
@@ -55,26 +60,42 @@ class ManualCurrentOracle(CurrentPriceOracleInterface):
 
     def __init__(self) -> None:
         super().__init__(oracle_name='manual current price oracle')
+        self.database: Optional['DBHandler'] = None
+
+    def set_database(self, database: 'DBHandler') -> None:
+        self.database = database
 
     def rate_limited_in_last(self, seconds: Optional[int] = None) -> bool:
         return False
 
-    def query_current_price(self, from_asset: Asset, to_asset: Asset) -> Price:
+    def query_current_price(
+            self,
+            from_asset: Asset,
+            to_asset: Asset,
+            match_main_currency: bool,
+    ) -> Tuple[Price, bool]:
         """
         Searches for a manually specified current price for the `from_asset`. If it finds
-        the price, converts it to a price in `to_asset` and returns.
+        the price and it is the main currency, returns as is and a boolean `True`.
+        Otherwise converts it to a price in `to_asset` and returns it + a boolean `False`.
```

Comment:
```suggestion
        the price and it is the main currency, returns as is and along with a True value to indicate that main currency price was used.
        Otherwise converts it to a price in `to_asset` and returns it along with a False value to indicate no main currency matching happened.
```